src/vertexAttribute.js

```diff
@@ -21,7 +21,8 @@ vgl.vertexAttributeKeys = {
   'Normal' : 1,
   'TextureCoordinate' : 2,
   'Color' : 3,
-  'CountAttributeIndex' : 4
+  'Scalar': 4,
+  'CountAttributeIndex' : 5
```

Comment:
Right, I believe the one in geomData was left over from a bad merge. Now you mentioned it, I remember it. This change looks good to me.